arrow-site/doc/docs/arrow-core/datatypes/optiont/README.md

```diff
@@ -44,7 +44,7 @@ fun getCountryCode(maybePerson : Option<Person>): Option<String> =
   }
 ```
 
-Nested flatMap calls flatten the `Option` but the resulting function starts looking like a pyramid and can easily lead to callback hell.
+Nested flatMap calls flatten the `Option`, but the resulting function starts looking like a pyramid and can easily lead to callback hell.
```

Comment:
This looks copy-pasted from `Either`. Someone should probably work on this sometime to avoid the repetition.